content/c/concepts/enums/enums.md

```diff
@@ -0,0 +1,126 @@
+# Enum
```

Comment:
This entry is missing the required metadata. To understand what to put here, please read the [metadata section in the style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#metadata). The metadata should look like this:

```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---

Also, we do not have h1 headers to title Doc entries, the title in the metadata does this automatically.